demo-terraform/aws/serverless-app/api-gateway.tf

```diff
@@ -1,28 +1,32 @@
 
 
 resource "aws_api_gateway_rest_api" "ride" {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure that the API endpoint type in API Gateway is set to private for API Gateway Rest API**
 
**Vulnerability**
API endpoint type in API Gateway is not set to private for API Gateway Rest API. This may lead to unauthorized access/data leak.
 
**Remediation**
Private API endpoints can be created using AWS console.
 In AWS Console - 
1. Sign in to the API Gateway console.
2. Select Create API.
3. Under REST API, select Build.
4. Enter a name for API Name.
5. For Endpoint Type, select Private.
6. Click on Create API.
 In Terraform - Add 'PRIVATE' to the list of 'endpoint_configuration.types'.

source: accurics-